components/brave_wallet/browser/keyring_service.cc

```diff
@@ -760,13 +761,13 @@ void KeyringService::RestoreWallet(const std::string& mnemonic,
   auto* keyring = RestoreKeyring(mojom::kDefaultKeyringId, mnemonic, password,
                                  is_legacy_brave_wallet);
   if (keyring && !keyring->GetAccountsNumber()) {
+    const std::string account_name = GetAccountName(1);
     const auto address =
-        AddAccountForKeyring(mojom::kDefaultKeyringId, GetAccountName(1));
+        AddAccountForKeyring(mojom::kDefaultKeyringId, account_name);
```

Comment:
why this change?

Reply:
It's leftover from when I was constructing AccountInfos which need account_name.  Will revert